### DISCUSSION TITLE
Customize ld-select selected options rendering

### DISCUSSION THREAD

**Discussion by dpitois** [emdgroup-liquid/liquid#633] [Q&A] [answered]
Hi!

I was looking for available options to customize how the selected options are rendered when using the ld-select (with multiple enabled).
Did not find another issue or question about that, if I missed it, sorry, and I will try to inspire from that.

My need is to render options in the popover using a LaTeX wrapper, and so far for now, I can put custom HTML inside the ld-option, the rendering is working as expected.
But, when we select an option (or more), it looks like only the `.innerText` part is taken and renderer in parent ld-select component.

Is there a way to take hand on how is rendered this selection?

Example: [https://stackblitz.com/edit/github-fvtkeq?file=src%2Fcomponents%2FForm%2FForm.tsx](https://stackblitz.com/edit/github-fvtkeq?file=src%2Fcomponents%2FForm%2FForm.tsx)  
I took the React sandbox, and _just_ add the following code to the theme option part in order to add a red square:
```jsx
<div
  style={{
    width: '1em',
    height: '1em',
    display: 'inline-block',
    backgroundColor: 'red',
    marginRight: '1em',
  }}
></div>
```
It looks OK for the popover:

![image](https://user-images.githubusercontent.com/7702006/231485531-1bce999e-0d04-4522-bacb-f15c1cf3b6bc.png)

but only the text part is visible on selection:

![image](https://user-images.githubusercontent.com/7702006/231485639-10594863-dbc1-447d-bbf2-89689d04c118.png)

**Comment by borisdiakur** [comment, marked as answer]
Hi @dpitois 
I have created a [feature request](https://github.com/emdgroup-liquid/liquid/issues/637) from your question and am working on a solution right now. Will let you know there, as soon as I have a solution.
🙌

**Comment by borisdiakur** [comment]
@dpitois It's working now with @emdgroup-liquid/liquid v5.2.0: https://stackblitz.com/edit/github-fvtkeq-dbreyw?file=package.json%3AL6
